sdk/src/main/java/com/silanis/esl/sdk/service/SigningStyleService.java

```diff
@@ -96,4 +99,37 @@ public void deleteSigningThemes() {
             throw new EslException("Could not update the signing themes to account.", e);
         }
     }
+
+    /**
+     * Save account signing logos.
+     *
+     * @param signingLogos
+     */
+    public void saveSigningLogos(List<SigningLogo> signingLogos) {
+        String path = template.urlFor(UrlTemplate.ACCOUNT_SIGNING_LOGO_PATH).build();
+        String payload = JacksonUtil.serialize(signingLogos);
+        try {
+            client.post(path, payload);
+        } catch (RequestException e) {
+            throw new EslServerException("Could not save the signing logos for account.", e);
+        } catch (Exception e) {
+            throw new EslException("Could not save the signing logos for account.", e);
+        }
+    }
+
+    /**
+     * Get account signing logos.
+     *
+     */
+    public List getSigningLogos() {
+        String path = template.urlFor(UrlTemplate.ACCOUNT_SIGNING_LOGO_PATH).build();
+        try {
+            String stringResponse = client.get(path);
+            return SigningLogoConverter.converToSDKSigningLogoList(Serialization.fromJsonToList(stringResponse, com.silanis.esl.api.model.SigningLogo.class));
+        } catch (RequestException e) {
+            throw new EslServerException("Could not get the signing logos from account.", e);
+        } catch (Exception e) {
+            throw new EslException("Could not get the signing logos from account.", e);
+        }
+    }
```

Comment:
This part has already been modified and doesn't need to be modified. Please rebase the dev branch.